react/BaseContent.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { ExtensionPoint } from 'vtex.render-runtime'
+import { useOrderForm } from 'vtex.order-manager/OrderForm'
+import { useCssHandles } from 'vtex.css-handles'
+import { Button } from 'vtex.styleguide'
+
+import { useMinicartState } from './MinicartContext'
+
+interface OrderFormContext {
+  orderForm: OrderForm
+  loading: boolean
+}
+
+interface Props {
+  sideBarMode: boolean
+  finishShoppingButtonLink: string
+}
+
+const CSS_HANDLES = [
+  'minicartContent',
+  'minicartFooter',
+  'minicartEmptyStateText',
+  'minicartEmptyStateContainer',
+] as const
+
+const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
+  const { orderForm, loading }: OrderFormContext = useOrderForm()
+  const handles = useCssHandles(CSS_HANDLES)
+  const { variation } = useMinicartState()
+
+  const sideBarStyles = {
+    height: '100%',
+  }
+
+  const popupStyles = {
+    maxHeight: 600,
+  }
```

Comment:
Move this outside of component. Right now it's instancing a new object on every render. Might be a silly saving, but 🤷‍♂ it doesnt cost anything

Actually, all those styles in `style` props could be defined using CSS Modules. So move static styles definitions to CSS modules.

Reply:
Nice! Agree that it would be better to use CSS modules for this